src/nickel.rs

```diff
@@ -341,31 +337,37 @@ impl<D: Sync + Send + 'static> Nickel<D> {
     // }
 }
 
-#[cfg(test)]
-mod tests {
-    use crate::Nickel;
-    use std::str::FromStr;
-    use std::net::SocketAddr;
+// The first test here is just testing the standard library. The other two
+// aren't currently supported. The second is kind of just a std lib test too.
 
-    #[test]
-    #[should_panic(expected = "invalid socket address")]
-    fn invalid_listen_addr() {
-        Nickel::new().listen("127.0.0.1.6667").unwrap();
-    }
+// TODO: migration cleanup - The last two tests are commented out since we no
+// longer provide access to the underlying server. Do we need to?
```

Comment:
... I've run into annoyances trying to shut down Hyper from a multi-threaded program.  It would be nice if Nickel allowed us to whack the server as part of clean shutdown, but the general practice of "let the thread die on program exit" is adequate, if not elegant.

This is a complicated way of saying I'd like access to the underlying server, but it's not critical.